protocol/flatbuffers/rpc.fbs

```diff
@@ -181,3 +188,31 @@ table AutoBoneEpochResponse {
     epoch_error: float32;
     adjusted_skeleton_parts: [SkeletonPart];
 }
+
+enum FkToggle: uint8 {
+    NONE = 0,
+    EXTENDED_SPINE_MODEL = 1,
+    EXTENDED_PELVIS_MODEL = 2,
+    EXTENDED_KNEE_MODEL = 3,
+    FORCE_ARMS_FROM_HMD = 4
+}
+
+table FkSkeletonToggle{
```

Comment:
It is not clear to me how this is supposed to be used. Why does a toggle have both an enum and a bool? Also a toggle doesn't have state associated with it - it toggles, if you meant "set the setting described by the enum to the value described by the bool", this should not say "toggle"

Reply:
We set a value of the enum to true or false...
What naming would you propose instead of toggle? Toggle can be "toggled on" or "toggled off"...